ui/js_utils/text_validation.js

```diff
@@ -93,7 +93,7 @@ export class TextValidation {
    */
   validate_platemap_editor_input(text) {
     let feedback = "";
-    const valid_regex = new RegExp("^[0-9A-Za-z _-]+$");
+    const valid_regex = new RegExp("^[0-9A-Za-z ./_-]+$");
```

Comment:
Does the period need to be escaped? I think otherwise it matches any character https://regexone.com/lesson/wildcards_dot

Reply:
not inside a character class

Uploading Screen Recording 2023-07-21 at 11.51.07 AM.mov…

Reply:
Ah cool, didn't realize